modules/benchmarks/src/largetable/render3/index.html

```diff
@@ -26,6 +26,7 @@ <h2>Render3 Largetable Benchmark</h2>
   </div>
 
   <script>
+    ngDevMode = false;
```

Comment:
Can we just build a minified code which removes the `ngDevMode` completely? That would be better than checking it everywhere.

Reply:
Reverted as it doesn't visibly impact benchmarks.
Let's take care of this once we dig into building and packaging render3 apps